src/Core/WikiDeclensions.fs

```diff
@@ -0,0 +1,50 @@
+﻿module WikiDeclensions
+
+open FSharp.Data
+open NounCategories
+open Article
+open StringHelper
+
+type EditableArticleOneDeclension = HtmlProvider<"https://cs.wiktionary.org/wiki/panda">
+type EditableArticleTwoDeclensions = HtmlProvider<"https://cs.wiktionary.org/wiki/čtvrt">
+type LockedArticle = HtmlProvider<"https://cs.wiktionary.org/wiki/debil">
+
+let getUrl = (+) "https://cs.wiktionary.org/wiki/"
+
+let getNumberOfDeclensions =
+    getContent
+    >> getChildPart "čeština"
+    >> getChildPart "podstatné jméno"
+    >> getChildrenPartsWhen (starts "skloňování")
+    >> Seq.length
+
+let getEditable case number word =
+    let numberOfDeclensions = getNumberOfDeclensions word
+    let url = getUrl word
+    match numberOfDeclensions with
+    | 1 ->
```

Comment:
So there're no nouns with >2 declensions, right?

Reply:
I have not seen and cannot imagine more than two declensions within one noun section, yet Czech is quite unpredictable...